### ISSUE TITLE
When an image fails to load, show alt text if available

### ISSUE THREAD

**Issue by Sjmarf** [mlemgroup/mlem#784]
Images in posts/comments are formatted as `![Alt text](https://google.com/myimage.png)`. If alt text is available, we should show it when the image fails to load.

**Comment by Sjmarf** [comment]
Possible child of #378

**Comment by d42ohpaz** [comment]
To add to this, I learned today you can [add hover text](https://lemm.ee/comment/18536628) to images by adding quoted text after the image url: 

`![alt text](https:://example.com/image.png "Hover text")`

I feel this would be a nice addition to this ticket, but if y'all feel this should be separate, I'm happy to create a new issue.

**Comment by Sjmarf** [comment]
At the moment we're using the presence of hover text to decide whether an image should be displayed as a full-size, tappable image or as an emoji. All custom emojis have hover text, and people rarely use hover text apart from emojis. So, we render all images with hover text as emojis. Unfortunately, there isn't really a better way to decide whether an image should be an emoji or not without using the tooltip value.
